mlir/python/mlir/dialects/affine.py

```diff
@@ -3,3 +3,141 @@
 #  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 from ._affine_ops_gen import *
+from ._affine_ops_gen import _Dialect, AffineForOp
```

Comment:
Nit: why do we need to import `AffineForOp` if we import `*` above?  (This is different for `_Dialect` that is private due to underscore in the name and is therefore not imported as part of `*`.)

Reply:
I think explicitly importing names used in the file is not a bad thing. I also think this is the natural result of essentially treating these files as `__init__.py`s for modules that don't exist structurally (i.e., it should be `affine/__init__.py`) _and_ as implementation files as well. Anyway I'll take it out.